ideation/README.md

```diff
@@ -1,4 +1,17 @@
 # How to come up with an idea
 
+Sometimes coming up with an idea for a hackathon can be really difficult. However, the good thing to keep in mind is that this is a project to be made in a limited amount of time so it doesn't have to be the perfect idea and can be something that you build off of in the future. It's a good opportunity to try and learn new things with little being risked. Because at hackathons you can work on anything that uses technology, like website/app, a video game, a VR/AR app, etc. there are endless options. 
 
+#### Narrowing down topics 
+What helps me is narrowing down some broader category I want to create a project in. Some hackathons will have a theme or categories you can submit too which can help narrow down the ideas. Others will have company sponsored topics that typically have some kind of specific project they are judging for. It can be helpful to look over these when you're looking for an idea.
 
+#### Create a pool of ideas
+Once you have narrowed down your topic, I like having everyone just pool in ideas and write them down. Typically this will lead to people jumping off each others ideas and in the end you will have a solid list of ideas. When thinking of ideas, it can be beneficial to think about problems that you have faced or that you know about that you want to address. The best ideas I've seen have typically been aimed at solving some sort of problem whether that be wanting to play songs at the same time with friends or wanting to better pair people up in airplane seats based on personality. 
+
+#### Narrow down your ideas
+Of these, some will be really bad, others will have a lot of potential. Go through your list and remove the ideas that your team doesn't like. Something that's very important is that everyone on the team likes the idea and is passionate about it. You won't be able to all work well together if people aren't happy with your final idea. Also think about how feasible the idea is. If everyone on your team needs to learn some new technology for this project, you probably don't want to have a super complicated project. It's much easier to create a minimal viable product (MVP) and then add on features and embellishments than it is to try to narrow a project down and cut out important features later on.
+
+#### Finaling deciding on a project
+After you have narrowed down your ideas to 2 or 3 ideas that everyone likes and thinks is doable, it can be helpful to make a pros and cons list. Or look into the technology you will need and start planning out how you would make each project. THis can expose hidden complexities that may help narrow down the choices. Sometimes all your ideas are equally good and in that case, just pick an idea. Flip a coin if needed! 
```

Comment:
```suggestion
After you have narrowed down your ideas to 2 or 3 ideas that everyone likes and thinks is doable, it can be helpful to make a pros and cons list. Alternatively, your team can look into the technology you will need and start planning out how you would make each project. This can expose hidden complexities that may help narrow down the choices. Sometimes all your ideas are equally good and in that case, just pick an idea. Flip a coin if needed! Remember that you can always change your project, and that there is no pressure to do anything :)
```